panel.html

```diff
@@ -14,7 +14,10 @@ <h1>Methods 2023 - Panel Questions</h1>
         <li>Cameron Wilcox - What is something you wish you knew when you got out of college?</li>
         <li>Kruize Christensen - What are some resources that we can study that would not have been taught at MSU in the field of computer science?</li>
         <li>Kruize Christensen - What are some beginning jobs to look at as a Computer Science graduate that are located in State?</li>
-        <li>Kruize Christensen - What makes you stand out to employers compared to other recent graduates?</li>        
+        <li>Kruize Christensen - What makes you stand out to employers compared to other recent graduates?</li>
+	<li>Darshan Diamond - If I have a website with a list of projects, what kind of projects do you like to see when concidering someone for a new hire?</li>
```

Comment:
concidering - spelling

Reply:
I fixed the spelling on that and pushed it up.